src/api/python/cvc5.pxi

```diff
@@ -3531,6 +3531,21 @@ cdef class Term:
         """:return: True iff this term is a tuple value."""
         return self.cterm.isTupleValue()
 
+    def isUninterpretedSortValue(self):
```

Comment:
This seems like it was accidentally added?

Reply:
Oops, I have no clue how that happened and how that didn't result in an error...